core/services/ardupilot_manager/mavlink_proxy/MAVLinkRouter.py

```diff
@@ -60,7 +60,7 @@ def convert_endpoint(endpoint: Endpoint) -> str:
         log = f"--log {self.logdir().resolve()}"
 
         if master.connection_type == EndpointType.TCPServer:
-            return f"{self.binary()} --tcp-port {master.argument} {log} {endpoints}"
+            return f"{self.binary()} --tcp-endpoint {master.place}:{master.argument} --tcp-port 0 {log} {endpoints}"
```

Comment:
Add a comment explaining about `tcp-port 0` and why is necessary

Reply:
Done.